Core/GDCore/IDE/Events/ProjectElementRenamer.cpp

```diff
@@ -64,34 +69,46 @@ class GD_CORE_API ExpressionIdentifierStringFinder
   void OnVisitNumberNode(NumberNode &node) override {}
   void OnVisitTextNode(TextNode &node) override {}
   void OnVisitVariableNode(VariableNode &node) override {
-    if (node.child)
-      node.child->Visit(*this);
+    if (node.child) node.child->Visit(*this);
   }
   void OnVisitVariableAccessorNode(VariableAccessorNode &node) override {
-    if (node.child)
-      node.child->Visit(*this);
+    if (node.child) node.child->Visit(*this);
   }
   void OnVisitVariableBracketAccessorNode(
       VariableBracketAccessorNode &node) override {
     node.expression->Visit(*this);
-    if (node.child)
-      node.child->Visit(*this);
+    if (node.child) node.child->Visit(*this);
   }
   void OnVisitIdentifierNode(IdentifierNode &node) override {}
   void OnVisitObjectFunctionNameNode(ObjectFunctionNameNode &node) override {}
   void OnVisitFunctionCallNode(FunctionCallNode &node) override {
     gd::String lastLayerName;
     gd::ParameterMetadataTools::IterateOverParametersWithIndex(
-        platform, globalObjectsContainer, objectsContainer, node,
+        platform,
+        globalObjectsContainer,
+        objectsContainer,
+        node,
         [&](const gd::ParameterMetadata &parameterMetadata,
             std::unique_ptr<gd::ExpressionNode> &parameterNode,
-            size_t parameterIndex, const gd::String &lastObjectName) {
+            size_t parameterIndex,
+            const gd::String &lastObjectName) {
           if (parameterMetadata.GetType() == "layer") {
-            // Remove quotes, it won't match if it's not a literal anyway.
-            lastLayerName = expressionPlainString.substr(
-                parameterNode->location.GetStartPosition() + 1,
-                parameterNode->location.GetEndPosition() -
-                    parameterNode->location.GetStartPosition() - 2);
+            if (parameterNode->location.GetEndPosition() -
```

Comment:
this part here is probably overkill for functions?
But it doesn't hurt checking we're not doing a substring that doesn't make sense